src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -180,6 +211,39 @@ public static Inet4Address ofLiteral(String ipv4AddressLiteral) {
         return parseAddressString(ipv4AddressLiteral, true);
     }
 
+    /**
+     * Creates an {@code Inet4Address} based on the provided {@linkplain
+     * Inet4Address##format textual representation} of an IPv4 address in POSIX form.
+     * This form allows extended syntax, accepting hexadecimal and octal address
+     * segments as specified in {@code inet_addr} POSIX API (loose syntax). Please
+     * refer to <a href="https://www.ietf.org/rfc/rfc6943.html#section-3.1.1">
+     * <i>RFC&nbsp;6943: Issues in Identifier Comparison for Security Purposes</i></a>.
+     * <p> If the provided IPv4 address literal cannot represent a {@linkplain
+     * Inet4Address##format valid IPv4 address} an {@code IllegalArgumentException} is thrown.
+     * <p> This method doesn't block, i.e. no hostname lookup is performed.
+     *
```

Comment:
Please desrcibe the syntax this metod accepts here, in the normative part of the specification.

Reply:
I would rather move the description of the loose syntax fully here, since the only method that accepts it is `Inet4Address.ofPosixLiteral`. We need to be absolutely clear that this is the only method that accepts it. Also, I feel that a link to an external document is not sufficient in this case. It's good to have it, but since we actually implement the parsing in java we should describe clearly the accepted syntax and how we parse it here. We have no control over external documents, they can move, their content can change, etc... I believe that the anchor `Inet4Address##format-posix` should lead to this method here, not to the class-level documentation. Having a short paragraph in the class level documentation that explains that an alternative loose syntax is accepted by a specific method is fine. However I don't feel that describing this loose syntax in the class level API documentation is a good idea.

Reply:
Moved the algorithm description from the class spec to method spec.